docs/docs/migration/migration-from-v6-to-v7.md

```diff
@@ -8,8 +8,86 @@ sidebar_position: 1
 First, execute `npx @comet/upgrade@latest v7` in the root of your project.
 It automatically installs the new versions of all `@comet` libraries, runs an ESLint autofix and handles some of the necessary renames.
 
+<details>
+
+<summary>Changes handled by @comet/upgrade</summary>
+
+-   Disabling GraphQL field suggestions
+-   Importing the types of `@comet/admin-theme` in `vendors.d.ts`
+-   Replacing the Roboto font with Roboto Flex
+
+</details>
+
 ## API
 
+### Remove unnecessary dependencies
+
+Remove following dependencies **if you don't use them in your project**:
+
+-   `@aws-sdk/client-s3`
+-   `@azure/storage-blob`
+-   `pg-error-constants`
+
+### Provide `strategyName` in createStaticCredentialsBasicStrategy
+
+Make sure to use a meaningful strategy name as this name can be used to identify the user when using this strategy more than once. Do not forget to add the strategy to the App Guard.
+
+```diff
+  createStaticCredentialsBasicStrategy({
+      password: "xxxxx",
++     strategyName: "system-user",
+  }),
+```
+
+```diff
+  {
+      provide: APP_GUARD,
+-     useClass: createCometAuthGuard(["static-credentials-basic", "..."]),
++     useClass: createCometAuthGuard(["system-user", "..."]),
+  };
+```
+
+```diff
+  UserPermissionsModule.forRootAsync({
+      useFactory: (...) => ({
++         systemUsers: ["system-user"],
+          ...
+      }),
+      ...
+  }),
+```
+
+### Remove `language` field from `User`
+
+```diff
+// static-users.ts
+
+export const staticUsers = {
+    admin: {
+        id: "3b09cc12-c7e6-4d16-b858-40a822f2c548",
+        name: "Admin",
+        email: "admin@customer.com",
+-       language: "en",
+    },
+    // ...
+} satisfies Record<string, User>;
+```
+
+### Remove `@PublicApi()` and rename `@DisableGlobalGuard()`
```

Comment:
We could add an upgrade script for that. But should we? Verifying if endpoints should even be public seems like a good idea to me.

Reply:
That was my thought too. I think it's a good security review